lib/takeaway.rb

```diff
@@ -0,0 +1,32 @@
+require 'twilio-ruby'
+# require_relative '../.env.rb'
+
+class Takeaway
+
+  def menu
+    @menu = { 'Tomato soup' => 5,
+              'Steak and chips' => 15,
+              'Salmon Stew' => 12,
+              'Chicken currey' => 10,
+              'Beef casserole' => 18 }
+  end
+
+  def calculate_delivery_time(time = Time.now)
+    @hour = time.hour + 1
```

Comment:
Why is this an instance variable? Instance variables are set on the instance, so are accessible for the lifetime of that instance (i.e. until the program is shut down). Nowhere else in your code needs to know about `hour`, so it can just be a regular plain variable.